installer/install.iss

```diff
@@ -3812,4 +3850,24 @@ begin
         if not DeleteFile(FileName) then
             LogError('Line {#__LINE__}: Unable to delete file "'+FileName+'". Please do it manually after logging off and on again.');
     end;
+
+    {
+        Remove posh-git
+    }
+
+    if (IsComponentSelected('poshgit')) then begin
+        Cmd:='"'+ExpandConstant('{sysnative}\WindowsPowerShell\v1.0\powershell.exe')+'"'+
+            ' -ExecutionPolicy Bypass -NoProfile -NoLogo -NonInteractive -WindowStyle Hidden -command "'+
+            'Uninstall-Package posh-git -Scope AllUsers -ErrorAction SilentlyContinue; ' +
+            '$res=$?; ' +
+            'if ($res) {' +
+            ' Remove-PoshGitFromProfile -AllUsers; ' +
```

Comment:
Since this function will be provided by the module, should we uninstall after it has been removed from profile?

Reply:
But of course :grin: